### PR TITLE
Fixes #16707: After upgrade from 5.0 to 6.0 on centos7, rudder-slapd is not started

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/16707

### CODE REVIEW

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -359,6 +359,24 @@ then
   systemctl stop rudder-slapd
 fi
 
+#=================================================
+# Post transaction
+#=================================================
+%posttrans -n rudder-webapp
+
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]: We can just run `systemctl start rudder-slapd` anyway

**Comment by ncharles** [comment]
PR updated with a new commit

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 2]
```diff
@@ -359,6 +359,14 @@ then
...
+# during upgrade, service may have been stopped by uninstall of rudder-inventory-ldap
+systemctl start rudder-slapd
```
Comment (amousset) [inline rudder-webapp/SPECS/rudder-webapp.spec]:
```suggestion
systemctl start rudder-slapd >/dev/null
```

**Comment by ncharles** [comment]
OK, squash merging this PR

**Pull request merged**